### PR TITLE
WIFI-2410: Added patch to host-apd to fix ubus assoc validation

### PR DESCRIPTION
**Pull request by shimmy568**
Signed-off-by: Owen Anderson <owenthomasanderson@gmail.com>

Fairly small change. Just moved the code to send the assoc response to be after the ubus validation code runs instead of before it. I was unable to test this code due to the fact that the auth response also checks rrsi levels as well as the WiFi client I was using which caused it to be impossible to test (at least with my knowledge). 

But it shouldn't break anything, just possibly prevent weird edge cases with roaming

### CODE REVIEW

**Review by RickSommerville — APPROVED** [review]
Looks good.

File: feeds/wifi-ax/hostapd/patches/601-fix_ubus_assoc.patch [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+-		reply_res = send_assoc_resp(hapd, sta, mgmt->sa, resp, reassoc,
+-					    pos, left, rssi, omit_rsnxe);
+-	os_free(tmp);
+-
+ 	ubus_resp = hostapd_ubus_handle_event(hapd, &req);
+ 	if (ubus_resp) {
+ 		wpa_printf(MSG_DEBUG, "Station " MACSTR " assoc rejected by ubus handler.\n",
+@@ -4576,6 +4571,11 @@ static void handle_assoc(struct hostapd_
+ 		goto fail;
+ 	}
```
Comment (yashvardhan-netex) [inline feeds/wifi-ax/hostapd/patches/601-fix_ubus_assoc.patch:17]: Can we move this even up? I mean why do the other stuff if we are not going to associate the client in case of low rssi.

File: feeds/wifi-ax/hostapd/patches/601-fix_ubus_assoc.patch [hunk 2]
```diff
@@ -0,0 +1,26 @@
+--- a/src/ap/ieee802_11.c
```
Comment (yashvardhan-netex) [inline feeds/wifi-ax/hostapd/patches/601-fix_ubus_assoc.patch:1]: I believe we should apply the patch on top of the latest patch. Sometimes the hunk succeeds but it might corrupt the other patches.

File: feeds/wifi-trunk/hostapd/patches/601-fix_ubus_assoc.patch [hunk 3]
```diff
@@ -0,0 +1,26 @@
+--- a/src/ap/ieee802_11.c
```
Comment (yashvardhan-netex) [inline feeds/wifi-trunk/hostapd/patches/601-fix_ubus_assoc.patch:1]: Same here. Please apply on top of the latest patch.

**Review by yashvardhan-netex — COMMENTED** [review]
I think we should still find a way to test it. Nevertheless, I hope we have tested the pass scenario, I mean testing the association with multiple clients. You may use lanforge for the same.

File: feeds/wifi-ax/hostapd/files/hostapd.sh [hunk 4]
```diff
@@ -214,6 +214,9 @@ hostapd_common_add_bss_config() {
 	config_add_int maxassoc max_inactivity
 	config_add_boolean disassoc_low_ack isolate short_preamble
 
+config_add_int signal_connect signal_stay signal_poll_time \
```
Comment (shimmy568) [inline feeds/wifi-ax/hostapd/files/hostapd.sh:217]: Ran into an issue on the wifi-6 ap's where we weren't applying the configuration for `signal_stay` and some other related fields to the hostapd configuration file. Making the client_disconnect_threshold field not work for wifi-6 aps.

**Comment by RickSommerville** [comment]
Merged 8a697710, thanks
